tests/functional/test_l10n.py

```diff
@@ -6,12 +6,12 @@
 
 import pytest
 
-from pages.home import HomePage
+from pages.firefox.new.download import DownloadPage
 
 
 @pytest.mark.nondestructive
 def test_change_language(base_url, selenium):
-    page = HomePage(selenium, base_url).open()
+    page = DownloadPage(selenium, base_url).open()
```

Comment:
I'm switching this functional test to point to `/firefox/new/` instead of the home page, as the test requires the same version of the language switcher to be available on all translated pages. With this PR the en-US home page now uses the Protocol lang switcher, but all other home page locales still use the Pebbles equivalent.